### ISSUE TITLE
Matching functionality of Apple News format?

### ISSUE THREAD

**Issue by adamkhan** [manton/JSONFeed#157]
It might be a good idea to consider matching the functionality of the Apple News format, which is also JSON. See the docs at https://developer.apple.com/documentation/apple_news/apple_news_format

**Comment by manton** [comment]
Anything in particular you'd like to see adopted from Apple News format? To be honest, I was disappointed that Apple invented a proprietary format here instead of continuing to build on RSS (or JSON Feed). It's very unlikely that any other apps are going to support Apple News.

**Comment by skyzyx** [comment]
I also thought that Apple News has removed support for publishing over RSS.

**Comment by adamkhan** [comment]
@manon Absolutely it's disappointing what Apple did, but they have created fields that help with the presentation/layout of feeds, which therefore seem to me worth considering for emulation. Of particular interest to me, as I'm building a feed reader, are the color fields.

Moreover, if JSONFeed was extended to include what Apple thinks it needs, then the company would have less excuse for not using JSONFeed. And if they ended up embracing JSONFeed, that might be a very nice boost for the format.

**Comment by j-f1** [comment]
It looks like (at first glance) Apple News Format is a replacement for HTML rather than RSS/JSON Feed.

**Comment by adamkhan** [comment]
@j-f1 First sentence in their overview:

> Apple News Format is the JavaScript Object Notation (JSON) format you use to create articles for Apple News.

**Comment by j-f1** [comment]
Yep! I looked through the docs and they seem to be all about formatting a single article, rather than syndicating a collection of articles. (although there is some overlap between Apple News files and JSON Feed Items)
